### DISCUSSION TITLE
"Please wait..." is displayed on the phone menu after firmware update. What's the reason?

### DISCUSSION THREAD

**Discussion by jeweitian** [Mr-MIBonk/M.I.B._More-Incredible-Bash#141] [Q&A] [answered]

**Comment by harman-f** [comment]
FW update 
or 
M.I.B install?

**Reply by harman-f** [reply]
Run the FW Update again. 
looks like something did not run OK. 

Did you get any errors during installation?

Always use a good SD card

**Comment by Jackie264** [comment, marked as answer]
Enter the RED menu then check your FECs
